craft_cli/helptexts.py

```diff
@@ -0,0 +1,291 @@
+# Copyright 2021 Canonical Ltd.
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License version 3 as published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with this program; if not, write to the Free Software Foundation,
+# Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+
+"""Provide all help texts."""
+
+import textwrap
+from operator import attrgetter
+from typing import TYPE_CHECKING, List, Tuple
+
+if TYPE_CHECKING:
+    from craft_cli.dispatcher import BaseCommand, CommandGroup
+
+
+# max columns used in the terminal
+TERMINAL_WIDTH = 72
```

Comment:
Why do we use 72 instead of, say, 78?

Reply:
I wanted to be conservative here. Note that this module does not present the texts to the user, just builds them and returns them. Maybe the caller would want to indent them one or two columns... so I decided not to use "80", and choose what PEP8 recommends for docstrings, which is "72".

But I'm to change it if you have reasons.

Reply:
I'm good with 72, I was just curious about this exact value.